components/03-modules/modal/modal--header-footer.hbs

```diff
@@ -0,0 +1,42 @@
+{{#if trigger }}
+    {{ render '@modal-trigger' modalTrigger }}
+{{/if}}
+<div role="dialog"
+     aria-hidden="true"
+     id="{{ modal.id }}"
+     data-modal="{{ modal.id }}"
+     class="modal {{ modal.class }}"
+     tabindex="-1"
```

Comment:
please remove tabindex, the `role="dialog"` make an element focusable

Reply:
done